src/plot_api/plot_schema.js

```diff
@@ -56,16 +56,17 @@ PlotSchema.get = function() {
     return plotSchema;
 };
 
-PlotSchema.crawl = function(attrs, callback) {
+PlotSchema.crawl = function (attrs, callback, specifiedLevel) {
+    var level = specifiedLevel || 0;
```

Comment:
As I had to get familiar with it, I can add docs. Caveat, the nearly identical crawler was already in place (in `plot_schema.js`) and I'm not the biggest `Plotly` schema expert.